app/src/app/profile/page.tsx

```diff
@@ -43,6 +43,13 @@ export default function Profile() {
       },
     });
 
+  const { data: marriages } = api.marriage.getMarriedUsers.useQuery(
+    {},
+    {
+      staleTime: 5000,
```

Comment:
I think we could set the stale-time for this to be significantly higher? Something like 5 minutes or so.